docs/matlab/tutorials/README.rst

```diff
@@ -0,0 +1,8 @@
+.. _examples_index:
```

Comment:
I think what you're trying to do is have this at the MATLAB top-level? If yes, if its moved to docs/matlab/index.rst (index.rst is the convention rather than README in Sphinx)and this file is added to the main index instead of matlab/api and matlab/tutorial then that should work.